src/main/java/storagesystem/viewcontroller/FrameworkController.java

```diff
@@ -54,7 +50,16 @@ public void initialize(URL location, ResourceBundle resources) {
     void userButtonPressed(){ loadUI("settings/userPage"); }
 
     @FXML
-    void teamButtonPressed(){ loadUI("settings/teamPage"); }
+    void teamButtonPressed(){
+        if(StoreIT.getCurrentOrganisation().getUsersTeams(StoreIT.getCurrentUser()).size() > 0){
+            loadUI("settings/teamPage");
+        } else{
+            NoTeamPopUpController popUp = new NoTeamPopUpController();
+            rootPane.getChildren().add(popUp);
+            AbstractFader.fadeTransition(popUp, 3);
```

Comment:
I'd say the animation is very quick now, barely enough time to read it, change to ~6-7sec